conda_build/_load_setup_py_data.py

```diff
@@ -111,7 +111,7 @@ def setup(**kw):
             exec(code, ns, ns)
     else:
         if not permit_undefined_jinja:
-            raise TypeError(f"{setup_file} is not a file that can be read")
+            raise TypeError("%s is not a file that can be read" % setup_file)  # noqa: UP031
```

Comment:
We need this old syntax because this module will be injected in the workdir to be executed by a potentially older Python (e.g. 3.5).

Reply:
Isn't this more like a ValueError or RuntimeError (or FileNotFoundError)?

Reply:
🤷 I didn't put it there :/